.travis.yml

```diff
@@ -4,3 +4,4 @@ node_js:
   - 'iojs'
   - '0.12'
   - '0.10'
+before_script: npm -g install npm@latest
```

Comment:
because of: https://github.com/npm/npm/blob/master/changelogs/CHANGELOG-1.md#v1429-2015-10-29